FRC1076-2016-Competition/src/org/usfirst/frc/team1076/robot/physical/Robot.java

```diff
@@ -127,6 +127,14 @@ public void robotInit() {
 		armFollower.enableBrakeMode(true);
 		armExtendMotor.enableBrakeMode(true);
 		armExtendFollower.enableBrakeMode(true);
+
+		
+		armMotor.ConfigFwdLimitSwitchNormallyOpen(true);
+		armMotor.ConfigRevLimitSwitchNormallyOpen(true);
+		armMotor.enableLimitSwitch(true, true);
+		armExtendMotor.ConfigFwdLimitSwitchNormallyOpen(true);
+		armExtendMotor.ConfigRevLimitSwitchNormallyOpen(true);
+		armExtendMotor.enableLimitSwitch(true, true);
```

Comment:
Ok. I'll try this in today's debug but if it doesn't work then we'll just keep the current code.